src/platform_impl/windows/drop_handler.rs

```diff
@@ -24,8 +26,8 @@ pub struct FileDropHandlerData {
     window: HWND,
     send_event: Box<dyn Fn(Event)>,
     cursor_effect: u32,
-    hovered_is_valid: bool, /* If the currently hovered item is not valid there must not be any
-                             * `HoveredFileCancelled` emitted */
+    enter_is_valid: bool, /* If the currently hovered item is not valid there must not be any
```

Comment:
```suggestion
    valid: bool, /* If the currently hovered item is not valid there must not be any
```

Maybe just like that should be fine?

Reply:
Renamed to `valid`.